apps/automation-v2/src/tlc/batch-liquidate.ts

```diff
@@ -60,9 +64,30 @@ export async function batchLiquidate(
     return chunk;
   };
 
+  let res: AxiosResponse<GetUserResponse> | undefined = undefined;
+  try {
+    // try subgraph call with thegraph api endpoint 
+    res = await getTlcUsers(
+      ctx,
+      config.SUBGRAPH_URL,
+      config.SUBGRAPH_RETRY_LIMIT
+    );
+  } catch (e) {
+    // log thegraph request api error
+    if (e instanceof SubgraphError){
+      ctx.logger.error(JSON.stringify(e));
```

Comment:
If it's another kind of error, would it be useful to still log that string representation (without json stringify I guess)?

Reply:
ctx logger doesn't log json objects properly, without stringify is logs `[object]` instead of the proper error msg, we could only log the `e.message`, but we will lose other important details, below it's an example of the full error:

```
[2023-11-30T03:11:50.330Z] error: {"cause":{"message":"Request failed with status code 404","name":"AxiosError","stack":"AxiosError: Request failed with status code 404\n    at settle (/Users/anon/Documents/Web3/temple/apps/automation-v2/dist/main.js:145018:12)\n    at BrotliDecompress.handleStreamEnd (/Users/anon/Documents/Web3/temple/apps/automation-v2/dist/main.js:145865:11)\n    at BrotliDecompress.emit (node:events:525:35)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","config":{"transitional":{"silentJSONParsing":true,"forcedJSONParsing":true,"clarifyTimeoutError":false},"adapter":["xhr","http"],"transformRequest":[null],"transformResponse":[null],"timeout":0,"xsrfCookieName":"XSRF-TOKEN","xsrfHeaderName":"X-XSRF-TOKEN","maxContentLength":-1,"maxBodyLength":-1,"env":{},"headers":{"Accept":"application/json, text/plain, */*","Content-Type":"application/json","User-Agent":"axios/1.5.1","Content-Length":"118","Accept-Encoding":"gzip, compress, deflate, br"},"method":"post","url":"https://api.thegraph.com/subgraphsdd/name/medariox/v2-mainnet","data":"{\"query\":\"{\\n        tlcUsers(block: {number: 18189826} where: {debt_gt: \\\"0\\\"}) {\\n          id\\n        }\\n      }\"}"},"code":"ERR_BAD_REQUEST","status":404},"name":"SubgraphError"}
```

Reply:
will add anyways the `ctx.logger.error(e)`, thx for the suggestion